roles/ceilometer-data/tasks/monitoring.yml

```diff
@@ -1,10 +1,5 @@
 ---
```

Comment:
similar as above.  Let's change this file to checks.yml and do the actual check installation here.